build.sbt

```diff
@@ -84,26 +98,64 @@ lazy val interop = crossProject(JSPlatform, JVMPlatform)
   .in(file("interop"))
   .settings(stdSettings("zio-interop"))
   .dependsOn(core % "test->test;compile->compile")
+  .settings(
+    scalacOptions in Test ++= Seq("-Yrangepos")
+  )
+
+lazy val interopJVM = interop.jvm
+
+lazy val interopJS = interop.js
```

Comment:
Should they be renamed to `interopSharedJVM` and `interopSharedJS`?

Reply:
Yes, probably, and placed in `interop-shared`.